go-controller/pkg/ovn/obj_retry.go

```diff
@@ -0,0 +1,129 @@
+package ovn
+
+import (
+	"reflect"
+	"sync"
+	"time"
+
+	"k8s.io/klog/v2"
+)
+
+// initRetryObj tracks a failed object to potentially retry later
+// initially it is marked as skipped for retry loop (ignore = true)
+func (oc *Controller) initRetryObj(obj interface{}, lock *sync.Mutex, retryObjs interface{}, key string) {
```

Comment:
retryObjs can still be a map[string]interface right? You don't need to make the entire map an interface?